PROJECT-CLEARWATER-MIB

```diff
@@ -4579,11 +4580,94 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
     DESCRIPTION "The total number of invalid timers processed over the period."
     ::= { chronosInvalidTimersProcessedEntry 2 }
 
+chronosTotalTimersTable OBJECT-TYPE
+    SYNTAX SEQUENCE OF ChronosTotalTimersEntry
+    MAX-ACCESS         not-accessible
+    STATUS             current
+    DESCRIPTION        "The total number of timers this Chronos node
+                       is managing - including primary and backup
+                       timers"

```

Comment:
Strictly speaking, "primary and backup timers" aren't well-defined - I think what you mean are both timers for which this node is primary, and for which this node is backup.